walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -384,23 +385,23 @@ Verify that you followed each step in the procedure above.  If you are still hav
 == Exploring data sync features using the showcase app
 
 To explore data sync features, run the multiple instances of the data sync app using different browsers.
-For example, use the browser on your mobile device as well as using the browser on your laptop.
+For example, use Chrome and Firefox browsers.
 
 image::images/showcase.png[showcase, role="integr8ly-img-responsive"]
 
 === Demonstrating real-time sync
 
-. On your laptop:
+. Using the Chrome browser:
 .. Select *Manage Tasks*.
 .. Create a new task using *+* icon.
 .. Enter some task text  and click *Create*.
 
-. On your mobile device:
+. Using the Firefox browser:
 .. Check that the same task appears in the *Manage Tasks* page.
 .. Make some changes to the task.
 
-. On your laptop:
-.. Check that the task changes are appear.
```

Comment:
Are we saying that the user has to have chrome and firefox installed on their mobile device?  If 2 different browsers are required, can we just leave the choice of them up to the user?\

Reply:
two browser from laptop/desktop is the only config supported.